src/pages/settings/Subscription/DisableAutoRenewSurveyPage.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import FeedbackSurvey from '@components/FeedbackSurvey';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+
+function DisableAutoRenewSurveyPage() {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const handleSubmit = () => {
+        // TODO API call to submit feedback will be implemented in next phase
+    };
+
+    return (
+        <ScreenWrapper
+            testID={DisableAutoRenewSurveyPage.displayName}
+            includeSafeAreaPaddingBottom={false}
```

Comment:
Is there any reason to set this to `false`? It doesn't looks good on iOS without the safe area padding.

![Simulator Screenshot - iPhone 15 Pro 17 2 - 2024-06-05 at 22 44 12](https://github.com/Expensify/App/assets/25520267/7091ccb1-72e0-496f-bd22-65e674250999)

Reply:
Could this in theory still be accessed on native via deep link? Should we implement `return null` for native?

Reply:
Maybe a better option than returning null would be to display `FullPageNotFoundView`?